File: ooplab/day1/1.cpp
#include <iostream>
using namespace std;

int main() {
  int x, y, temp;
  cout << "Enter first number: ";
  cin >> x;

  cout << "Enter second number: ";
  cin >> y;
  
  cout << "Before swapping x = " << x << " y = " << y << endl;

  temp = x;
  x = y;
  y = temp;

  cout << "After swapping x = " << x << " y = " << y;
  return 0;
}

File: pylab/day6/hw/1.py
import math

a = int(input("Enter first number: "))
b = int(input("Enter second number: "))

gcd = math.gcd(a, b)
lcm = math.lcm(a, b)

print(f"GCD of two numbers is: {gcd}")
print(f"LCM of two numbers is: {lcm}")